compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/CodeAnalyzer.java

```diff
@@ -2782,6 +2784,11 @@ public void visit(BLangExpressionStmt exprStmtNode) {
         this.checkStatementExecutionValidity(exprStmtNode);
         analyzeExpr(exprStmtNode.expr);
         validateExprStatementExpression(exprStmtNode);
+        BLangExpression exprStmt = exprStmtNode.expr;
```

Comment:
Also shouldn't this be
```suggestion
        BLangExpression expr = exprStmtNode.expr;
```